web/src/js/components/Header/CaptureMenu.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from "react";
+
+CaptureMenu.title = "Capture";
+
+export default function CaptureMenu() {
+    //empty component since we don't have an header menu for capture, but we just display the title
```

Comment:
also here, the comment

Reply:
probably here a comment would be useful since it's the only tab with an empty header. Someone might ask why in the future (?)